src/sessions/SessionsSection.tsx

```diff
@@ -21,7 +29,17 @@ const Clients = (row: UserSessionRepresentation) => {
 };
 
 export const SessionsSection = () => {
+  const { t } = useTranslation("sessions");
   const adminClient = useAdminClient();
+  const [filterDropdownOpen, setFilterDropdownOpen] = useState(false);
+  const [filterType, setFilterType] = useState(
+    t("sessionsType.allSessions").toString()
+  );
```

Comment:
There is no need to call to `toString()` here if the value is returned from a factory function.
```suggestion
  const [filterType, setFilterType] = useState(() => t("sessionsType.allSessions"));
```